.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "group": "test",
+      "label": "Run all tests",
+      "type": "shell",
+      "command": "bundle exec rake test",
+      "problemMatcher": [],
+      "presentation": {
+        "clear": true
+      }
+    },
+    {
+      "group": "test",
+      "label": "Run tests for current file",
+      "type": "shell",
+      "command": "bundle exec rake test TEST=${file}",
+      "problemMatcher": [],
+      "presentation": {
+        "clear": true
+      }
+    }
+  ]
+}
```

Comment:
and again :)

Reply:
And this file allows you to run the tests for the whole code base, or just the file you're on, from within the VSCode editor. We have a similar config for VSCode users in the monolith.